src/backend/gpopt/translate/CTranslatorDXLToPlStmt.cpp

```diff
@@ -180,13 +180,13 @@ CTranslatorDXLToPlStmt::GetPlannedStmtFromDXL(const CDXLNode *dxlnode,
 	// collect unique RTE in FROM Clause
 	List *oids_list_unique = NIL;
 
-	ListCell *lc_rte = nullptr;
+	ListCell *lc = nullptr;
```

Comment:
This renaming looks irrelevant to the patch. It is not a part of original commit. Why do we need it?